.github/workflows/cypress.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         with:
           repository: nextcloud/server
           submodules: true
-          ref: ${{ env.BRANCH }}
+          ref: bugfix/noid/remove-files-duplicate-dom-ids
```

Comment:
?

Reply:
this makes CI run against the branch from https://github.com/nextcloud/server/pull/33320

once https://github.com/nextcloud/server/pull/33320 is merged I'll revert this back